### PR TITLE
Update projects

### LINKED ISSUE #8: Add minima

### PR DESCRIPTION
**Pull request by DirtyF**
- [x] add `minima` (fix #8)
- [x] add `jekyll-commonmark`
- [x] add `jekyll-admin`
- [x] Sort plugins (GitHub Pages first)

### CODE REVIEW

**Comment by mattr-** [comment]
LGTM, based on my limited Go knowledge.

File: project.go [hunk 1]
```diff
@@ -10,27 +10,29 @@ var (
 	defaultProjectMap map[string]*Project
 	defaultProjects   = []*Project{
 		newProject("jekyll", "jekyll/jekyll", "master", "jekyll"),
-		newProject("jemoji", "jekyll/jemoji", "master", "jemoji"),
-		newProject("mercenary", "jekyll/mercenary", "master", "mercenary"),
-		newProject("jekyll-import", "jekyll/jekyll-import", "master", "jekyll-import"),
-		newProject("jekyll-feed", "jekyll/jekyll-feed", "master", "jekyll-feed"),
+		newProject("jekyll-admin", "jekyll/jekyll-admin", "master", "jekyll-admin"),
+		newProject("minima", "jekyll/jekyll-minima", "master", "minima"),
```
Comment (parkr) [inline project.go]: I think the repo is just jekyll/minima!

**Comment by DirtyF** [comment]
@parkr I'll fix the sorting, should we add all plugins used by GitHub Pages?

**Comment by DirtyF** [comment]
Order is now similar to https://pages.github.com/versions/ for GitHub Pages plugin, then plugins, then non plugins

**Comment by parkr** [comment]
> should we add all plugins used by GitHub Pages?

Nah, only ones the Jekyll organization owns.

File: project.go [hunk 2]
```diff
@@ -10,27 +10,29 @@ var (
...
+		newProject("jekyll-commonmark", "jekyll/jekyll-commonmark", "master", "jekyll-commonmark"),
 		newProject("jekyll-textile-converter", "jekyll/jekyll-textile-converter", "master", "jekyll-textile-converter"),
-		newProject("jekyll-redirect-from", "jekyll/jekyll-redirect-from", "master", "jekyll-redirect-from"),
-		newProject("github-metadata", "jekyll/github-metadata", "master", "jekyll-github-metadata"),
+		newProject("jekyll-admin", "jekyll/jekyll-admin", "master", "jekyll-admin"),
+		newProject("classifier-reborn", "jekyll/classifier-reborn", "master", "classifier-reborn"),
+		newProject("mercenary", "jekyll/mercenary", "master", "mercenary"),
 		newProject("plugins website", "jekyll/plugins", "master", ""),
-		newProject("jekyll-archives", "jekyll/jekyll-archives", "master", "jekyll-archives"),
-		newProject("jekyll docker", "jekyll/docker", "master", ""),
```
Comment (pathawks) [inline project.go]: `”jekyll docker”` is removed?

Comment (parkr) [reply project.go]: We don't have that repo anymore.

Comment (pathawks) [reply project.go]: Right ✅

**Pull request merged**